docs/metrics.rst

```diff
@@ -141,7 +140,10 @@ in it:
     one blue, cell and wifi tag get added. The tags depend on the
     number of valid :term:`stations` for each of the three.
 
+*Removed April 2020*: The ``region`` tag
```

Comment:
Sphinx has some additional directives that show nicely in the docs. There's `.. versionadded::`, `.. versionchanged::`, and `.. deprecated::`.

I think I'd use versionchanged here.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded